ACCESSIBILITY.md

```diff
@@ -0,0 +1,74 @@
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## 1.1 Core Principles
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to be able to participate because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of mangagmenet - developing a collective commons is 
+
+## 2.1 Web Accessibility
+
+[ORGANIZATION OR PROJECT NAME] is committed to ensuring the accessibility of this site for people with disabilities. We pledge to meet [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/), Level AA conformance. Any issues should be reported to [EMAIL ADDRESS].
+
+We are committed to ensuring accessibility through:
+
+- Use of ARIA landmark roles.
+- Proper use of HTML headings.
+- Perceptible link text and focus states.
+- Sufficient color contrast.
+- The use of appropriate alt text for images.
+- Keyboard navigable forms inputs with descriptive labels.
+- Text transcripts for audio and closed captioning for video content.
+- Navigation without the need for a mouse or track pad.
+
+### 2.2 Community Accessibility
+
+- Describing images & illustrations (Scriberia)
+- Understanding the way in which language is used
+- Cognitive load & in context of processing disabilities
+- Accessibility for beginners -> can link to Community Handbook (avoiding idioms)
+- Translation & localisation 
+- Writing good practices for data visualisation (not assuming particular IDEs, language-agnostic)
+
+### 3.1 How we work with Access-related Feedback
+Access-related feedback may be submitted via this anonymised form.
+
+1. Feedback form
+2. Drop-in sessions - 3x a year: drop-in session to receive feedback aabuout access related different parts of project. Discussion of practices
+3. Direct email to RCM with feedback for 1:1
+4. Drop by Accessibility Working Group meeting -> can focus on feedback
+
+Feedback is taken to monthly meeting with chairs of working groups & project leadership.
+
+## Core Principles
+1. We do these accessibility practices to remove barriers for people who historically have been limited in their ability to participate in because these projects benefit everyone
+2. Openness is a means to enable people with broad backgrounds to get involved. Openness for openness' sake does that erase this fact.
+3. Access to commons is a question of management - developing a collective commons is 
+
```

Comment:
```suggestion
```

Duplicated from above, see line 13

Reply:
Wow - this was an old version! Glad you got to see this - but we've updated the principles according to feedback to: Removing barriers, Sharing openly, and Supporting inclusive participation